src/org/parosproxy/paros/core/scanner/HostProcess.java

```diff
@@ -757,6 +765,20 @@ public void alertFound(Alert alert) {
             pluginStats.incAlertCount();
         }
         alertCount++;
+        
+        ExtensionAlert extensionAlertRef = Control.getSingleton().getExtensionLoader().getExtension(ExtensionAlert.class);
+        // only check if extension is loaded
+        if (extensionAlertRef != null) {

```

Comment:
The `ExtensionActiveScan` depends on `ExtensionAlert` so we can assume that it's loaded/enabled here.